adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java

```diff
@@ -132,7 +132,7 @@ public AdditionalInfo getInfo(final Principal clientIdentity) {
 
         if (clientIdentity instanceof PreSharedKeyIdentity) {
             final Span span = newSpan("PSK-getDeviceIdentityInfo");
-            final PreSharedKeyDeviceIdentity deviceIdentity = getHandshakeIdentity(clientIdentity.getName());
+            final PreSharedKeyDeviceIdentity deviceIdentity = getHandshakeIdentity(span, clientIdentity.getName());
```

Comment:
can you please adjust the change period in the copyright header as well?

Reply:
The period is specified as `LOWER, UPPER`, i.e. `2019, 2020` ...

Reply:
Adapted to
2019, 2020
2018, 2020